docs/changelog/86398.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 86398
+summary: User API actions allow use of internal usernames
```

Comment:
This will be in the release notes. So I'd avoid using "internal usernames" because (1) End-users likely won't understand what it means; and (2) We don't want to (unintentionally) encourage the usage of internal usernames. So can we have something more generic like: "Slightly relax username restrictions for User APIs"?

Reply:
Sounds good! We document internal users here: https://www.elastic.co/guide/en/elasticsearch/reference/current/internal-users.html but it's true that for most end-users this will be a foreign concept. +1 on not encouraging `_xpack` users for now reason 😅 

I will go with "Relax username restrictions for User APIs"